packages/flutter_tools/lib/src/linux/linux_doctor.dart

```diff
@@ -56,6 +56,7 @@ class LinuxDoctorValidator extends DoctorValidator {
     'gtk+-3.0',
     'glib-2.0',
     'gio-2.0',
+    'blkid',
```

Comment:
The message we show when anything here is missing was written with the assumption that installing GTK-dev libraries would install everything in the list as a dependency, so doesn't list the individual libraries. Do we need to revisit that string? E.g., on Fedora is it relatively easy to get into a state where of the things on this list only `blkid` is missing?

Reply:
In the particular case of `blkid`, if you don't explicitly grab it's location from pkg-config, it is implicitly assumed to be installed in the system root (see the linked bug).

This is not a problem if you manually install your GTK development packages locally (as instructed in that message), but it is a problem with the `flutter-snap` for example.

It's also not good enough just to have `libblkid` available in the snap. Without the project explicitly asking pkg-config where it is, the linker just makes guesses.

I don't think it' hurts to have `flutter doctor` check for this library in addition to the others, but if you think it unnecessary, we can take it off the list.

Reply:
In short, the message **doesn't** need to be updated, nor does this library need to be on this checklist.

Really, the only necessary changes in this PR are the `pkg_check_modules()` and `target_link_libraries()` additions in `flutter/CMakeLists.txt`.

Reply:
I'm fine with adding it to the list since it is in fact a dependency, I just wanted to be sure we didn't also need updated instructions. Thanks for the explanation; I understand now why we don't.